tests/test_packages.py

```diff
@@ -11,6 +11,10 @@
 from detection_rules.packaging import PACKAGE_FILE, Package
 
 
+with open(PACKAGE_FILE) as f:
+    package_configs = yaml.safe_load(f)['package']
```

Comment:
what do you think about making a `Package.get_configs()` function that encapsulates this?